plugins/package-managers/node/src/main/kotlin/Npm.kt

```diff
@@ -24,11 +24,6 @@ package org.ossreviewtoolkit.plugins.packagemanagers.node
 import java.io.File
```

Comment:
> since only a single package was processed

hm, I made the observation when running the fun tests, so not a single package but multiple package were involved. (A single package cannot be processed in parallel, so I guess this is a mis-understanding)

> and the parsePackage function is only called sequentially

I in fact verified that `getRemotePackageDetails()` executes only sequentally. But, `parsePackage()` is the only caller of that one, so I guess the same applied to `parsePackage()`.

What change to the commit message do you suggest?

Reply:
From what I understand, there are multiple aspects:
* The `withContext` construct in `getRemotePackageDetailsAsync` is only exited after all coroutines started within it are finished. Therefore, the `async` invocations are useless, since the function waits until the asynchronous processing is complete.
* But even on a higher level, no parallelization is applied; `parsePackage` is invoked sequentially for each package and waits for the result. So, even if asynchronous processing worked, it would not help here.

Do you agree with this?

Regarding the commit message, it is probably not necessary to describe this in detail. So, your text is probably fine.

Reply:
> Do you agree with this?

Yes, makes sense to me.